packages/i18n/src/locales/en_US.js

```diff
@@ -974,6 +974,18 @@ const translations = {
               example: 'db.coll.getName()',
               parameters: {}
             },
+            renameCollection: {
+              link: 'https://docs.mongodb.com/manual/reference/method/db.collection.renameCollection/',
+              description: 'Renames a collection.',
+              example: 'db.coll.renameCollection("newName")',
+              parameters: {}
+            },
+            runCommand: {
+              link: '',
+              description: 'Runs a db command with the given name where the first param is the collection name.',
+              example: '',
```

Comment:
`help` will look weird without an example. How about something like this: 
```
db.collection.runCommand( "text", { search: "searchKeywords"})
```

Reply:
Thanks!